Felienne/hedy
# Handle teacher requests through database and admin view instead of mails

**Description**
In this PR we change the flow of how we handle teacher requests. Currently these requests are sent as plain text to the admin to verify and then manually set these users as a teacher in the database. This is quite cumbersome and hard to test locally as we requires the mail services for validation. We change this to storing a `teacher_request` boolean on the profile, which will be removed when the request is accepted / rejected.

In alignment with this PR it might be nice to create an admin overview page containing some general info, such as the amount of pending requests; programs created and new accounts in the last day. 

**Fixes**
This PR fixes #3169.

**How to test**
Create a new account and request a teacher account. Verify that this is correctly stored in your local database and you can set this user as a teacher in the admin users view.
